api-server/src/server/boot/authentication.js

```diff
@@ -188,3 +194,53 @@ function createGetPasswordlessAuth(app) {
     );
   };
 }
+
+function mobileLogin(app) {
+  const {
+    models: { User }
+  } = app;
+  return async function getPasswordlessAuth(req, res, next) {
```

Comment:
Since this is async, express's error handling can't work properly (not express 4's, anyway!).  Could you wrap the whole thing in a try/catch block and call next(err)?